app/schemas/post.py

```
@@ -15,9 +17,19 @@ class PostCreate(PostBase):
 class PostUpdate(PostBase):
     pass
 
-class PostOut(PostBase):
+class PostResponse(PostBase):
     id: int
+    created_at: datetime
+    updated_at: datetime
+    view_count: int
 
     class Config:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)